docker-compose-dev.yml

```diff
@@ -27,6 +28,9 @@ services:
     volumes:
       - ./docker-dev-volumes/app/data:/dv
       - ./docker-dev-volumes/app/secrets:/secrets
+      # Uncomment for changes to xhtml in Netbeans to be deployed immediately.
```

Comment:
Could we comment a bit more broader for none netbeans users? Something like:
`Uncomment for changes to xhtml files on your local development file system to be mounted, read and deployed immediately to the container.`

Reply:
I guess but @poikilotherm made it sound like what Netbeans does is somewhat unique.

What if I change it to this?

`# Uncomment for changes to xhtml to be deployed immediately (if supported your IDE or toolchain).`

Reply:
@johannes-darms I didn't hear an objection so I switched to my suggestion: 5fa3f46ec6